etc/systemd/system/chia-monitor.service

```diff
@@ -0,0 +1,134 @@
+#   sudo touch /var/log/chia-monitor.log
+#   sudo cp etc/rsyslog.d/chia-monitor.conf /etc/rsyslog.d/
+#   sudo systemctl restart rsyslog
+#
+# Execute systemd commands for chia-monitor.service
+#   sudo cp chia-monitor.service /etc/systemd/system/
+#   sudo systemctl daemon-reload
+#   sudo systemctl start chia-monitor
+#   sudo systemctl status chia-monitor
+#   <if service is running without errors then enable in next step>
+#   sudo systemctl enable chia-monitor
+#
+# Check to see if port is open and running as chia-monitor user
+#   sudo netstat -lntp | grep 8000
+#   <example output>:
+#   tcp        0      0 0.0.0.0:8000            0.0.0.0:*               LISTEN      3672604/python
+#
+# Check procs to ensure chia-monitor user is owner of the previously identified process on port 8000
+#   sudo ps auxf | grep chia-monitor | grep -v grep
+#   <example output>:
+#   chia-mo+ 3672604  1.0  0.0 883468 88568 ?        Ssl  Aug31   6:20 /opt/chia-monitor/.local/share/virtualenvs/chia-monitor-kBBgivIG/bin/python -m monitor
+#
+# Finally, ensure service is operating as expected (example output below)
+#   systemctl status chia-monitor
+#      ● chia-monitor.service - chia-monitor
+#       Loaded: loaded (/etc/systemd/system/chia-monitor.service; enabled; vendor preset: disabled)
+#       Active: active (running) since Tue 2021-08-31 17:40:20 PDT; 9h ago
+#     Main PID: 3672604 (python)
+#       Tasks: 4 (limit: 1649800)
+#       Memory: 76.0M
+#       CGroup: /system.slice/chia-monitor.service
+#               └─3672604 /opt/chia-monitor/.local/share/virtualenvs/chia-monitor-kBBgivIG/bin/python -m monitor
+#     Sep 01 03:30:00 pipenv[3672604]: 2021-09-01T03:30:00.315 INFO   🟡 Pool Found Last 24H: 3410
+#     Sep 01 03:30:00 pipenv[3672604]: 2021-09-01T03:30:00.315 INFO   🟢 Pool Acknowledged Last 24H: 3409
+#     Sep 01 03:30:00 pipenv[3672604]: 2021-09-01T03:30:00.316 INFO   ❌ Pool Errors 24h: 0
+#     Sep 01 03:30:00 pipenv[3672604]: 2021-09-01T03:30:00.341 INFO   📶 Full Node Peer Count: 8
+#     Sep 01 03:30:00 pipenv[3672604]: 2021-09-01T03:30:00.342 INFO   📶 Farmer Peer Count: 1
+#     Sep 01 03:30:00 pipenv[3672604]: 2021-09-01T03:30:00.342 INFO   📶 Harvester Peer Count: 1
+#     Sep 01 03:30:01 pipenv[3672604]: 2021-09-01T03:30:01.296 INFO   ---------------------------
+#
+# Ensure Prometheus can access the chia-monitor exporter via firewall
+#   <exec firewall command for TCP port 8000 according to your deployment: iptables, firewall-cmd, etc>
+#   sudo firewall-cmd --permanent --zone=public --add-port 8000/tcp
+#   sudo firewall-cmd --reload
+#
+# Check log file to ensure functionality as expected
+#   sudo tail -f /var/log/chia-monitor.log
+#   <output should be consistent status at INFO level>
+#
+# Setup logrotate config for log file to avoid clogging up disk space
+#   sudo cp /opt/chia-monitor/etc/logrotate.d/chia-monitor /etc/logrotate.d/chia-monitor
+#   sudo sed -i '\:/var/log/chia-monitor.log:d' /etc/logrotate.d/syslog
```

Comment:
This step failed on my system as the `/etc/logrotate.d/syslog` file did not exist. Is it really necessary?

Reply:
adding the chia-monitor log file to rsyslog's logrotate job file isn't a requirement, just a nice to have on RH systems. it can be removed from the PR